styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  background-image: url(file:///C:/Users/Juan/Documents/GitHub/Portfolio/Images/background.png);
+  background-size: cover;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  text-align: center;
+  color: #42526e;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.headline {
+  text-align: center;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: bolder;
+  font-size: 19px;
+  margin-top: 100px;
+}
+
+.pheadline {
+  font-family: sans-serif;
+  font-size: 16px;
+  font-weight: bold;
+  color: #13397c;
+  margin: 1px 15px 1px 15px;
+}
+
+.mediaimgs {
+  display: flex;
+  justify-content: space-between;
+  margin: 50px 80px 1px 80px;
+}
+
+#p1 {
```

Comment:
- [ ] Please don't use ID as a selector.  If you attach a style to an ID, you won't be able to reuse it within the same webpage. Classes, however, can appear on several HTML elements on the same page. Being able to reuse styles is one of the advantages of CSS

Reply:
Well resolved

Reply:
Thank you @rloterh